resource_management_mobile/src/app/pages/resource-hiring/resource-hiring.page.scss

```diff
@@ -0,0 +1,62 @@
+@import "../../../theme/theme.scss";
```

Comment:
Rather than importing everywhere, Import in angular.json -> styles:[...here]

Reply:
Done